packages/amplication-client/src/Entity/ImportPrismaSchema/EntitiesImport.scss

```diff
@@ -63,4 +63,11 @@ $max-width: 700px;
     text-align: center;
     margin: var(--default-spacing);
   }
+
+  &__link-button {
+    @include body2;
+    a {
+      color: var(--dark-secondary-turquoise);
```

Comment:
@iamshabell thanks for your contribution.
We recently pushed a big UI change to the 'next' branch which may impact your PR. 
Please pull the recent changes from `next` into your branch and make sure you use the latest CSS variables and components.